apps/web/src/modules/auction/components/CurrentAuction/PlaceBid.tsx

```diff
@@ -116,7 +119,9 @@ export const PlaceBid = ({ highestBid, tokenId }: PlaceBidProps) => {
           <ContractButton
             className={auctionActionButtonVariants['bid']}
             handleClick={handleCreateBid}
-            disabled={address && !chain?.unsupported ? !isMinBid || !bidAmount : false}
+            disabled={
+              address && wagmiChain?.id == chain.id ? !isMinBid || !bidAmount : false
```

Comment:
strict equals? also the combination of || and ternary is slightly confusing at first glance. could we pull this out into a var and add a bit more clarity here?

Reply:
updated this check to be more clear + strict eq